cyclops-ctrl/go.mod

```diff
@@ -30,7 +30,6 @@ require (
 require (
```

Comment:
@kom-senapati can you try deleting the go.mod file from this line down, and then run `go mod tidy`?

Reply:
Some versions were updated

Reply:
You can push it to this branch. We don't want changes in the go.mod since nothing changes in the go code

Reply:
Reverted these 2 files to latest commits